privacy-security-explainer.md

```diff
@@ -86,7 +85,7 @@ However, visible-blurred should not be viewed as a substitute for TrustedUI and
 
 
 ### Feature policy
-**TODO** Fill this in with what is agreed upon in [#308](https://github.com/immersive-web/webxr/issues/308), [#729](https://github.com/immersive-web/webxr/issues/729), [#730](https://github.com/immersive-web/webxr/issues/730), and [#731](https://github.com/immersive-web/webxr/issues/731).
+All features in core WebXR module are controlled by the [feature policy "xr."](https://github.com/immersive-web/webxr/issues/729)
```

Comment:
The feature policy picked was `"xr-spatial-tracking"`

Reply:
ah gotcha